lib/widgets/payment_request_info_dialog.dart

```diff
@@ -82,34 +82,13 @@ class PaymentRequestInfoDialogState extends State<PaymentRequestInfoDialog> {
   Widget _buildPaymentRequestTitle() {
     return widget.invoice.payeeImageURL.isEmpty
         ? null
-        : Container(
-            height: widget.invoice.payeeImageURL.isEmpty ? 64.0 : 128.0,
-            padding: widget.invoice.payeeImageURL.isEmpty
-                ? EdgeInsets.zero
-                : EdgeInsets.only(top: 48.0),
-            child: Stack(
-              children: <Widget>[
-                Center(
-                  child: CircularProgressIndicator(
-                    valueColor: AlwaysStoppedAnimation<Color>(
-                      Theme.of(context).primaryTextTheme.button.color,
-                    ),
-                    backgroundColor: Theme.of(context).backgroundColor,
-                  ),
-                ),
-                Center(
-                    child: ClipOval(
-                  child: FadeInImage(
-                      width: 64.0,
-                      height: 64.0,
-                      placeholder: MemoryImage(widget._transparentImage),
-                      image: AdvancedNetworkImage(widget.invoice.payeeImageURL,
-                          useDiskCache: true),
-                      fadeOutDuration: Duration(milliseconds: 200),
-                      fadeInDuration: Duration(milliseconds: 200)),
-                )),
-              ],
-            ));
+        : Padding(
```

Comment:
Don't we want the loader experience as was before? Or the BreezAvatar already have this effect?

Reply:
BreezAvatar has this effect.